samples/eShopLite/ServiceDefaults/Extensions.cs

```diff
@@ -56,6 +57,12 @@ public static IHostApplicationBuilder ConfigureOpenTelemetry(this IHostApplicati
                 tracing.AddAspNetCoreInstrumentation()
                        .AddGrpcClientInstrumentation()
                        .AddHttpClientInstrumentation();
+            })
+            .ConfigureResource(rb => {
```

Comment:
I agree with the sentiment but honestly I am not sure if this can be done without some refactoring of OTLP configuration APIs. For now I just added it to the app templates.